### PR TITLE
chore: graphene upgrade

### PR DESCRIPTION
**Pull request by josebui**
## Description

- graphene upgrade
- Changed how mutation errors ar handled

### Related Issues
Fixes #....

### CODE REVIEW

File: requirements/base.in [hunk 1]
```diff
@@ -1,4 +1,4 @@
-graphene-django===2.15.0
+graphene-django===3.0.0
```
Comment (paulschreiber) [inline requirements/base.in:1]:
```suggestion
graphene-django
```

File: terraso_backend/apps/graphql/schema/commons.py [hunk 2]
```diff
@@ -25,7 +26,25 @@ def resolve_total_count(self, info, **kwargs):
...
+        abstract = True
+
+    errors = GenericScalar()
+
+    @classmethod
+    def mutate(cls, root, info, input):
+        try:
+            return super().mutate(root, info, input)
+        except Exception as error:
+            logger.error(
```
Comment (paulschreiber) [inline terraso_backend/apps/graphql/schema/commons.py:40]: question: do we want to use `logger.exception`?

**Review by paulschreiber — CHANGES_REQUESTED** [review]

File: terraso_backend/apps/graphql/schema/commons.py (hunk 2 above)
Comment (josebui) [reply terraso_backend/apps/graphql/schema/commons.py:40]: @paulschreiber I think that is a good idea to get the stacktrace and know the specific mutation easier

**Review by paulschreiber — APPROVED** [review]

File: terraso_backend/tests/graphql/mutations/test_group_associations_mutations.py [hunk 3]
```diff
@@ -62,9 +63,10 @@ def test_group_associations_add_by_non_parent_manager_fails(client_query, groups
         },
     )
     response = response.json()
+    print(response)
```
Comment (david-code) [inline terraso_backend/tests/graphql/mutations/test_group_associations_mutations.py:66]:
```suggestion
```

**Review by david-code — APPROVED** [review]
Looks good!

**Pull request merged**